requirements.txt

```diff
@@ -1,2 +1,3 @@
-django==4.0
-djangorestframework
\ No newline at end of file
+Django==4.0
+djangorestframework==3.13.1
+python-decouple==3.6
```

Comment:
I have seen some posts about the need to "pip uninstall decouple" and only than install python-decouple. So here it was not necessary? Has the docker python interpreter recognized it?

Reply:
I think it's not necessary since you don't installed the "decouple" module, what is not the correct one. The correct is called "python-decouple". When I did this task I had not configured the docker interpreter yet, so the only thing I did was install the python-decouple in my virtual environment and regenerate the requirements file by pip freeze. But the code is working properly, even with the docker interpreter, and the the problem with imports disappeared here.

Reply:
great, thanks. Approved.